app/components/talent-report.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  displaySubtitle: false
```

Comment:
since we dont need a subtitle at all, we can remove this property, and the usage on the template